### ISSUE TITLE
RTL language formatting

### ISSUE THREAD

**Issue by saud-97** [tachiyomiorg/tachiyomi#7017]
### Steps to reproduce

The formatting of RTL and LTR strings is not consistent across the app.

Inside download queue
![image](https://user-images.githubusercontent.com/39028181/165244434-9e6ab900-9fc5-42a3-9bc4-a187a27037ac.png)

Inside manga page
![image](https://user-images.githubusercontent.com/39028181/165244458-6d649e79-bd99-4f1c-9130-592a9b6e7b04.png)

inside the browse + extensions tab
![image](https://user-images.githubusercontent.com/39028181/165244464-f0809b8a-fb4d-4649-9757-62cb0d29a1ff.png)

![image](https://user-images.githubusercontent.com/39028181/165244795-e41c5f10-dc6e-4613-b9b2-05baa208ee95.png)

### Expected behavior

The RTL and LTR formatting should follow the main language of the app. Since my app is English it should follow English formatting LTR. 

### Actual behavior

It is mixing RTL and LTR formatting.

**Comment by arkon** [comment]
It doesn't make sense to treat an RTL string as a LTR string, would it? This seems consistent with what I've usually seen in apps/the web.

**Comment by saud-97** [comment]
Sorry if you misunderstood what I was referring to as RTL formatting but my point is it to "align" them correctly so they are both aligned to the left for consistency. Maybe my wording was a bit confusing.

What is happening here is that only parts of the contents become RTL. I believe it would make since to keep it RTL if the complete app became RTL depending on device language preferences. For example if you have file manager and a directory with RTL and LTR names you would always find them aligned to the same side no matter the direction. 

Basically what I'm thinking is the view table format look-alike where each column and row follows the one before it consistently.

**Comment by saud-97** [comment]
I can provide more detailed examples and explanation why it would make since to keep the alignment based on the device language not based on word direction if needed.

**Comment by arkon** [comment]
Wait, these screenshots aren't even from the main app.

I can at least verify that an Arabic source name is left-aligned in the Sources and Download Queue views.
